addons/account/wizard/account_payment_register.py

```diff
@@ -1000,3 +1008,11 @@ def action_open_untrusted_bank_accounts(self):
             }
 
         return action
+
+    def action_open_missing_account_partners(self):
+        self.ensure_one()
+        vals = {}
+        if len(self.missing_account_partner) > 1:
+            listview_id = self.env.ref('account.partner_missing_account_list_view').id
```

Comment:
not sure why you'd need a new list view here? was it specified to have only the name in list?

Reply:
See comment above

Reply:
See comment above

Reply:
ok

Reply:
ok ok don't be mad at me :sweat_smile: